docs/8.6.0_docs.md

```diff
@@ -73,7 +73,7 @@ When deployed within code, init is called before the graph/diagram description.
 ```
 
 **for example**:
-```
+```mermaid-code
```

Comment:
Should we use `mermaid-code` or `mermaid` ?
`mermaid` seems to be inline with other tags like `svelte`, `ts`, `bash`, etc.

Reply:
`mermaid` is already used to render diagrams, so I left that for rendering diagrams and kept `mermaid-code` for the syntax highlighting.

Reply:
Oh sorry, forgot about that. 😅

Saw that you had started https://github.com/github/linguist/issues/5578, so which keyword are you planning to use in there?

Reply:
For now, I was planning on just highlighting `.mmd` files, but I will need to research how to add markdown code fencing languages.

Reply:
So the use cases would be like this?
- `mermaid` -> Diagram
- `mermaid-code` -> Highlighted code
- `mermaid-example` -> Code + Diagram

The current usage in gitlab is 
<img width="654" alt="image" src="https://user-images.githubusercontent.com/10703445/139810892-ff0c3200-2918-4d5d-a497-ffa5ca731afa.png">

So I don't think we can change the usage of `mermaid`.

Should also get inputs from @knsv  and @mermaid-js/atlantians

Reply:
GitLab didn’t have a syntax highlighter for mermaid. I was thinking, if we actually do implement this in places like GitHub Linguist, code editors, and GitLab, we will have to be backwards compatible and making `mermaid` have syntax highlighting instead of rendering a diagram won’t be backwards compatible. Also, how you said above that languages like svelte, typescript, and bash have code fence names without the -code suffix, I was thinking maybe we can change `mermaid-code` to `mmd` (the file extension for mermaid’s files) and keep `mermaid-example` to be something local to mermaid’s internal documentation.